### PR TITLE
Profile paddle ocr

### PR DESCRIPTION
**Pull request by sfc-gh-dkajtoch**
* Added profiling utils for PaddleOCR
* Added sample pdf for testing

### CODE REVIEW

File: profile/profile_paddleocr.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    for idx in range(len(timeit_dpi.dpi_values)):
+        print(
+            f"DPI: {timeit_dpi.dpi_values[idx]} | "
+            f"{timeit_dpi.mean_exec_time[idx]:.4f} +/- "
+            f"{timeit_dpi.std_exec_time[idx]:.4f}"
+        )
+
+    # settings = PaddleSettings(det=True, rec=True, cls=False)
+    # cprofile_paddle = CprofilePaddle(settings)
+    # cprofile_paddle.profile()
```
Comment (sfc-gh-lduhr) [inline profile/profile_paddleocr.py]: Could we clean the comments? Either remove them or add a conditional call.

File: profile/profile_paddleocr.py [hunk 2]
```diff
@@ -0,0 +1,120 @@
+from paddleocr import PaddleOCR
+from dataclasses import dataclass, asdict
+
+from PIL.Image import Image as ImageType
```
Comment (sfc-gh-lduhr) [inline profile/profile_paddleocr.py]: This is a bit confusing. Why do we need to rename `Image`?

**Review by sfc-gh-lduhr — APPROVED** [review]

File: profile/profile_paddleocr.py (hunk 1 above)
Comment (sfc-gh-dkajtoch) [reply profile/profile_paddleocr.py]: I added click to select the mode

File: profile/profile_paddleocr.py (hunk 2 above)
Comment (sfc-gh-dkajtoch) [reply profile/profile_paddleocr.py]: cleaned up

**Review by sfc-gh-lduhr — APPROVED** [review]

**Pull request merged**